packages/SwingSet/test/change-parameters/bootstrap-change-parameters.js

```diff
@@ -0,0 +1,31 @@
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+
+export function buildRootObject() {
+  let vatAdmin;
+  let carolRoot;
+  let carolAdmin;
+
+  return Far('root', {
+    async bootstrap(vats, devices) {
+      vatAdmin = await E(vats.vatAdmin).createVatAdminService(devices.vatAdmin);
+    },
+
+    async prepare() {
+      // build Carol, the vat being changed
+      const bcap = await E(vatAdmin).getNamedBundleCap('carol');
+      const res = await E(vatAdmin).createVat(bcap);
+      carolRoot = res.root;
+      carolAdmin = res.adminNode;
+      await E(carolRoot).doSomething();
+    },
+
+    change() {
+      E(carolAdmin).changeOptions({
+        foo: 47,
```

Comment:
with vat-vat-admin doing checking, this `foo` will trigger an error.. we should test both a successful change and a rejected one